prime-dotnet-webapi/Controllers/SubmissionsController.cs

```diff
@@ -362,6 +362,45 @@ public async Task<ActionResult> AssignToaAgreementType(int enrolleeId, [FromBody
             return Ok(updatedEnrollee);
         }
 
+        // POST: api/enrollees/5/submissions/change/agreement-type
+        /// <summary>
+        /// Assign a TOA agreement type to the latest submission.
+        /// </summary>
+        /// <param name="enrolleeId"></param>
+        /// <param name="data"></param>
+        [HttpPost("{enrolleeId}/submissions/change/agreement-type", Name = nameof(ChangeToaAgreementType))]
```

Comment:
Just a style concern:  Should this be `HttpPut` and the path `{enrolleeId}/submissions/latest/agreement-type`?

Reply:
Updated to use PUT.